### ISSUE TITLE
Keep losing connection to SeedSync serivce

### ISSUE THREAD

**Issue by cogger12** [ipsingh06/seedsync#3]
Recently when ever I try and use seedsync it keeps randomly crashing. I have tried figuring it by myself but cant seem too. here is the debug log of the crash.
`2018/01/26 16:40:13 - ERROR - seedsync.ControllerJob - Caught exception in job ControllerJob
Traceback (most recent call last):
File "site-packages/pexpect/expect.py", line 99, in expect_loop
File "site-packages/pexpect/pty_spawn.py", line 462, in read_nonblocking
pexpect.exceptions.TIMEOUT: Timeout exceeded.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
File "python/common/job.py", line 44, in run
File "python/controller/controller_job.py", line 29, in execute
File "python/controller/controller.py", line 170, in process
File "python/controller/controller.py", line 280, in __update_model
File "python/lftp/lftp.py", line 245, in status
File "python/lftp/lftp.py", line 82, in wrapper
File "python/lftp/lftp.py", line 100, in __run_command
File "site-packages/pexpect/spawnbase.py", line 321, in expect
File "site-packages/pexpect/spawnbase.py", line 345, in expect_list
File "site-packages/pexpect/expect.py", line 107, in expect_loop
File "site-packages/pexpect/expect.py", line 70, in timeout
pexpect.exceptions.TIMEOUT: Timeout exceeded.
<pexpect.pty_spawn.spawn object at 0x7fb298092be0>
command: /usr/bin/lftp
args: ['/usr/bin/lftp', '-p', '22', '-u', 'cxxxx,', 'sftp://62.2xx.xxx.230']
buffer (last 100 chars): b' '
before (last 100 chars): b' '
after: <class 'pexpect.exceptions.TIMEOUT'>
match: None
match_index: None
exitstatus: None
flag_eof: False
pid: 2310
child_fd: 26
closed: False
timeout: 30
delimiter: <class 'pexpect.exceptions.EOF'>
logfile: None
logfile_read: None
logfile_send: None
maxread: 2000
ignorecase: False
searchwindowsize: None
delaybeforesend: 0.05
delayafterclose: 0.1
delayafterterminate: 0.1
searcher: searcher_re:
0: re.compile("b'lftp cxxxxx@62.2xx.xxx.230:.*>'")
2018/01/26 16:40:13 - DEBUG - seedsync.ControllerJob - Calling cleanup for ControllerJob
2018/01/26 16:40:13 - DEBUG - seedsync.Controller - Exiting controller
2018/01/26 16:40:13 - INFO - seedsync - Exiting Seedsync
2018/01/26 16:40:13 - DEBUG - seedsync.Controller.MPLogger - Stopped listener thread
2018/01/26 16:40:13 - INFO - seedsync.Controller - Exited controller
2018/01/26 16:40:13 - DEBUG - seedsync.ControllerJob - Finished cleanup for ControllerJob
2018/01/26 16:40:13 - DEBUG - seedsync.ControllerJob - Thread ControllerJob stopped`

I removed my IP just cause I dont want ever one knowing it.

Thanks.

**Comment by ipsingh06** [comment]
I can't reproduce problem on my end. Can you try the latest version? If you experience the same problem there, try the docker image.

**Comment by ipsingh06** [comment]
I'm closing this issue for now. Please create a new issue if you still experience this in the newest verison.
